stripe/res/values/styles.xml

```diff
@@ -146,6 +146,19 @@
         <item name="android:textColor">@color/stripe_paymentsheet_textinput_color</item>
     </style>
 
+    <style name="StripeBillingAddressCountryCell">
```

Comment:
maybe `StripeBillingAddressCountryItem`?

Reply:
Actually renamed to `StripePaymentSheetCountryItem` as it's only used in `PaymentSheet`, before it using `StripePaymentSheetTextInputEditText` with some additional attributes